src/js/smooch.jsx

```diff
@@ -161,6 +164,14 @@ export class Smooch {
                 }
             });
         }).then((loginResponse) => {
+            observable.on('socket:connected', () => {
```

Comment:
I'm not trying to cut corners, I just genuinely can't ever think of the correct path to take when I'm coding in redux. I find it extremely confusing.